### ISSUE TITLE
possibility to omit tests

### ISSUE THREAD

**Issue by Fabian188** [anishida/lis#27]
We build lis automatically for our open source software.

For this purpose it would be convenient to have the possibility to configure lis to omit tests. Indeed the tests created (minor) issues in the past for us.

Thank you,

Fabian

**Comment by anishida** [comment]
Fabian,

Thank you for your comment. Do you mean the tests performed by 'make check'? Could you give us the details on your issues?

Sincerely,
Akira Nishida

**Comment by Fabian188** [comment]
I do only "make" and "make install". Is there a make option build only the lib? That would do the job.

The issue was, that on the build runner find, diff and cmp were not installed, then one gets the error

```
 libtool: link: f95 -fopenmp -o esolve etest5.o  -L../src /builds/cfs/CFS/build/cfsdeps/lis/src/lis-build/src/.libs/liblis.a -lm -fopenmp
/usr/bin/ld: etest5.o: in function `main':
etest5.c:(.text.startup+0x18): undefined reference to `lis_initialize'
/usr/bin/ld: etest5.c:(.text.startup+0x34): undefined reference to `lis_printf'
/usr/bin/ld: etest5.c:(.text.startup+0x4a): undefined reference to `lis_printf'

```

**Comment by anishida** [comment]
Fabian,

Thank you for your pointing out. We released 2.0.26, in which you can disable building test programs by specifying option --disable-test. I appreciate it if you could check it. 

Sincerely,
Akira Nishida

**Comment by Fabian188** [comment]
Akira,

seems to work perfectly. Thanks!

Fabian
